pkg/grpc/auditlogstore/audit_log_store.go

```diff
@@ -0,0 +1,88 @@
+package auditlogstore
+
+import (
+	"time"
+
+	"github.com/square/p2/pkg/audit"
+	audit_log_protos "github.com/square/p2/pkg/grpc/auditlogstore/protos"
+	"github.com/square/p2/pkg/logging"
+	"github.com/square/p2/pkg/store/consul/transaction"
+
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+)
+
+type AuditLogStore interface {
+	List() (map[audit.ID]audit.AuditLog, error)
+	Delete(
+		txn *transaction.Tx,
+		id audit.ID,
+	) error
+}
+
+type store struct {
+	auditLogStore AuditLogStore
+	logger        logging.Logger
+	txner         transaction.Txner
+}
+
+func New(auditLogStore AuditLogStore, logger logging.Logger, txner transaction.Txner) audit_log_protos.P2AuditLogStoreServer {
+	return store{
+		auditLogStore: auditLogStore,
+		logger:        logger,
+		txner:         txner,
+	}
+}
+
+func (s store) List(_ context.Context, _ *audit_log_protos.ListRequest) (*audit_log_protos.ListResponse, error) {
```

Comment:
right now we just always list all of the records, so the list request doesn't inform the server how to do anything. we might care about it if we want to set a max number of records or something like that